pkg/cmd/run.go

```diff
@@ -134,8 +135,9 @@ func RunMicroshift(cfg *config.Config) error {
 		klog.Fatalf("MicroShift must be run privileged")
 	}
 
-	klog.InfoS("MICROSHIFT STARTING")
 	microshiftStart := time.Now()
+	startRec := *startuprecorder.New()
+	startRec.LogMicroshiftStart(microshiftStart)
```

Comment:
How about we use names without the `Log`? It's a "Recorder" struct, so it's already hinting we're recording these events
- MicroShiftStarts
- ServicesStart
- MicroShiftReady

Reply:
Yeah I was also thinking of doing this, good point, changed